demos/app.js

```diff
@@ -19,9 +24,50 @@ const app = module.exports = express({
 	viewsDirectory: '/demos',
 	layoutsDir: 'demos',
 	partialsDirectory: process.cwd(),
-	directory: process.cwd()
+	directory: process.cwd(),
+});
+
+const templatePartialDirectory = path.join(__dirname, '../templates/partials');
+fs.readdirSync(templatePartialDirectory).forEach(filename => {
+	handlebars.registerPartial(
+		`templates/${filename.substr(0, filename.lastIndexOf('.'))}`,
```

Comment:
[Copy/Pasta] It looks like this should switch places with line 42 to become `templates/partials/...`

Reply:
good spot